a4c-brooklyn-plugin/pom.xml

```diff
@@ -111,7 +83,22 @@
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-software-database</artifactId>
             <version>${brooklyn.version}</version>
-            <scope>test</scope>
+        </dependency>
+
+        <dependency>
```

Comment:
weird visualization! I'm pretty sure I added this dependency now, and not moved from anywhere. Anyway, I agree it should have <scope>test</scope>